plan/config.go

```diff
@@ -57,12 +59,19 @@ func ReadConfigFromEnviron() (*Config, error) {
 			behaviors[behavior.Name] = behavior
 		}
 	}
+
+	jsonReportPath := os.Getenv(jsonReportPathKey)
+	if jsonReportPath == "" {
+		jsonReportPath = "/crossdock/report.json"
```

Comment:
Should we make the environment variable required instead? No JSON reporting if the report path isn't set? Sending something to `/crossdock` is undesirable for when you're running it directly outside of docker.